index.html

```diff
@@ -342,6 +342,11 @@ <h2>General rules</h2>
 			<li><abbr title="User Interface Automation">UIA</abbr>: expose as <code>AriaRole</code> property. The <code>AriaRole property</code> can also support secondary roles using a space as a separator.</li>
 			<li><abbr title="Accessibility Toolkit">ATK</abbr>/<abbr title="Assistive Technology - Service Provider Interface">AT-SPI</abbr>: expose as an object attribute pair (<code>xml-roles:&quot;string&quot;</code>)</li>
 		  </ul>
+    </section>
+		<section id="roleMappingComputedRole">
+		  <h2>Computed Role</h2>
+                  <p>The computed role of an element is a string that represents the role of the element for the purposes of specification comformance testing. When an element has a role but is not contained in the required context (for example, an role `listitem` outside of a `list`), the computed role of the element is unspecified.</p>
```

Comment:
maybe even say "used on an element lacking an accessibility parent with a role of `list`".

```suggestion
                  <p>The computed role of an element is a string that represents the role of the element for the purposes of specification comformance testing. When an element has a role but is not contained in the required context (for example, a role `listitem` used on an element outside of a `list`), the computed role of the element is unspecified.</p>
```